cloudformation/campaign-central-backups.yaml

```diff
@@ -0,0 +1,175 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Campaign central databases backups.
+Parameters:
+  SourceTableName:
+    Description: DynamoDB tableName to backup.
+    Type: String
+    Default: campaign-central-PROD-campaigns
+  DestinationTableName:
+    Description: DynamoDB tableName to write to.
+    Type: String
+    Default: campaign-central-DEV-campaigns
+Resources:
+  CampaignCentralDynamoDBBackup:
+    Type: "AWS::DataPipeline::Pipeline"
+    Properties:
+      Name: CampaignCentralDynamoDBBackup
+      Description: "Pipeline to backup DynamoDB data to S3"
```

Comment:
`... to backup PROD DynamoDB data to DEV` ?

Reply:
The template is parameterised, so it could be any source table to any destination table!